networks/rhs.H

```diff
@@ -266,6 +267,12 @@ template<int n1, int n2>
 AMREX_GPU_HOST_DEVICE AMREX_INLINE
 constexpr int is_jacobian_term_used ()
 {
+
+#ifndef STRANG
```

Comment:
Can this be done now for both Strang and simplified SDC?

Reply:
technically no, practically, maybe.

The issue is that we are sparse in terms of d(Xdot, edot)/d(X, T), but when we switch to d(Xdot, edot)/d(X, e), we are dense -- there are effectively no zeros, because of the effect of how T changes with X when e is held constant.

Reply:
but I suspect that dT/dX |_e is small, so it may be that we can ignore this.